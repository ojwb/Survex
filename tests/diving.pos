( Easting, Northing, Altitude )
(    0.00,     0.00,     0.00 ) 1
(    0.00,     1.00,    -0.00 ) 2
(    1.00,     1.00,    -0.00 ) 3
(    1.00,     1.00,     1.03 ) 3a
(    1.00,     0.00,     0.00 ) 4
(    0.00,     0.00,     0.00 ) 5
(    0.00,     1.00,    -0.00 ) 6
(    1.00,     1.00,    -0.00 ) 7
(   -3.00,     1.00,    -3.00 ) 8
(   -3.00,    13.00,    -8.00 ) 9
(    3.93,    13.00,   -12.00 ) 10
