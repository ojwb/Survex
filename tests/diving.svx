; pos=yes warn=1
*fix 1 0 0 0
*data diving from to fromdepth todepth tape compass
1 2 0.0 0.0 1.00 000
2 3 0.0 0.0 1.00 090
3 3a 0.0 1.1 1.00 180

*data diving to from fromdepth todepth tape compass
4 3 1.0 1.0 1.00 180
5 4 1.0 1.0 1.00 270

*data diving to from depthchange tape compass
6 5 0.0 1.00 000
7 6 0.0 1.00 090

*data diving from fromcount fromdepth compass to tocount todepth
; 3:4:5 triangle...
7	123	5.0	270	8	128	2.0

*data diving station count depth newline compass
; 5:12:13 triangle...
8	128	2.0
			000
9	141	-3.0

; check diving leg with clino
*data diving from to fromdepth todepth tape compass clino
9 10 0 -4.0 8.0 090 -30
