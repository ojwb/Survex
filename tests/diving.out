./diving.svx:6:14: warning: Tape reading is less than change in depth
 3 3a 0.0 1.1 1.00 180
              ^~~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 11 survey stations, joined by 10 legs.
There are 0 loops.
Total length of survey legs =   33.03m (  33.03m adjusted)
Total plan length of survey legs =   28.93m
Total vertical length of survey legs =   13.03m
Vertical range = 13.03m (from 3a at 1.03m to 10 at -12.00m)
North-South range = 13.00m (from 10 at 13.00m to 5 at -0.00m)
East-West range = 6.93m (from 10 at 3.93m to 9 at -3.00m)

There were 1 warning(s).
